### PR TITLE
test(perf): update configuration of k6, prometheus and grafana dashboard

### PR DESCRIPTION
**Pull request by ytvnr**
## Issue

https://gravitee.atlassian.net/browse/APIM-609

## Description

- prometheus: lower scrape_interval and evaluation_interval to 1s (and add a comment in provisioning-datasources.yaml)
- k6 command: add prometheus rw trend options to export p99 p95 p90 and avg data
- grafana: create a k6_rate_interval used by k6 oriented widget, fix the widgets when needed

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-vlnzkwtkhu.chromatic.com)

🚀 CI was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.z6.web.core.windows.net/apim-609-update-k6-dashboard/index.html)
_Notes_: The deployed app is linked to the management API of APIM master. (Same login and password as APIM master)

### CODE REVIEW

File: gravitee-apim-perf/grafana/config/apim-performances.json [hunk 1]
```diff
@@ -327,38 +352,37 @@
...
-          "expr": "http_req_failed",
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "k6_http_reqs_total{expected_response=\"false\"}",
           "format": "time_series",
+          "hide": false,
           "instant": false,
-          "interval": "",
-          "legendFormat": "",
+          "legendFormat": "{{status}}",
```
Comment (leleueri) [inline gravitee-apim-perf/grafana/config/apim-performances.json:379]: Not sure if status is enough, we may have status 0 and in that case, the cause is in the error label... don't know if the label can be adapt based on a condition. but let's keep it like this for now

File: gravitee-apim-perf/grafana/config/apim-performances.json [hunk 2]
```diff
@@ -295,15 +295,41 @@
         "type": "prometheus",
         "uid": "PBFA97CFB590B2093"
       },
-      "description": "",
+      "description": "Number of failure over time",
```
Comment (leleueri) [inline gravitee-apim-perf/grafana/config/apim-performances.json:298]: I was expecting to keep the Global counter and add another widget with errors over time. wdyt ?

**Review by leleueri — APPROVED** [review]

File: gravitee-apim-perf/grafana/config/apim-performances.json (hunk 2 above)
Comment (ytvnr) [reply gravitee-apim-perf/grafana/config/apim-performances.json:298]: indeed we can, I feel the information was useless when having both of them.
As this PR aims to fix the dashboard I would put more effort during our "analyze" oriented issues.
Maybe we could inspire from something like https://grafana.com/grafana/dashboards/16543-k6-load-testing-results-prometheus/

wdyt ?

File: gravitee-apim-perf/grafana/config/apim-performances.json (hunk 1 above)
Comment (ytvnr) [reply gravitee-apim-perf/grafana/config/apim-performances.json:379]: indeed that would be interesting, but not sure we can apply conditions.

**Pull request merged**